### DISCUSSION TITLE
USB understanding problem

### DISCUSSION THREAD

**Discussion by nekton2** [ElectroTechnique/TSynth-Teensy4.1#133] [T4.1 code development]
The TSynth receives power via J4, USB Client and starts.  If the TSynth connected directly to the PC, the TSynth device is also available. When I connect J3, USB Host to the PC nothing happens.  I misunderstood something? Or is something wired incorrectly?

How works J3 = USB MIDI In? An external keyboard with USB MIDI out can be plugged in here? But there is no connection possible between PC and TSynth? Only via J4?

Alongside: TSynth does not start with power via J3.  If the TSynth is started with power via J4, it will not turn off if it continues to be powered via J3.

**Comment by Electrotechnique** [comment]
Power to TSynth is via USB Client (J4) only. You cannot power it via the USB Host. This isn't how USB 2.0 works.

**Reply by nekton2** [reply]
OK, thanks. And J3 is only USB MIDI In? For an external keyboard, but no PC?

**Reply by Electrotechnique** [reply]
Yes, it's for USB MIDI only, you cannot apply power to the synth with this - the USB 2.0 specification doesn't allow this anyway.
The USB Client socket is for power, MIDI IN/Out from a PC and Digital Audio to a PC.
